Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLoggerManager.cs

```diff
@@ -104,7 +104,7 @@ public ExecutionLoggerConfiguration Configuration
                                 RunSetReport.Name = defaultRunTabLogName;
                                 mLogsFolderName = defaultRunTabLogName + "_" + mCurrentExecutionDateTime.ToString("MMddyyyy_HHmmss");
                             }
-                            mExecutionLogger.ExecutionLogfolder = mConfiguration.ExecutionLoggerConfigurationExecResultsFolder + "\\" + mLogsFolderName + "\\" + this.GingerData.Seq.ToString() + " " + this.GingerData.GingerName + "\\";
+                            mExecutionLogger.ExecutionLogfolder = Path.Combine(mConfiguration.CalculatedLoggerFolder,mLogsFolderName,this.GingerData.Seq.ToString() + " " + this.GingerData.GingerName + "\\");
```

Comment:
"\\" can't be used in Linux, way we need them if doing PAth.Combine in all places?